src/frontend/src/utils/types.ts

```diff
@@ -0,0 +1,29 @@
+export type Team = {
+    id: string,
+    name: string,
+    type: string,
+    total_score: number
+}
+
+export type History = {
+    [key: string]: number
+}
+
+export type Scenario = {
+    id: number,
+    name: string
+}
+
+export type Result = {
+    id: string,
+    scenario: number,
+    state: "FAILED" | "DONE" | "PENDING"
+    score: number,
+    average_time: number
+}
```

Comment:
```suggestion
export type Result = {
    id: string,
    scenario: number,
    status: "FAILED" | "DONE" | "PENDING"
    score: number,
    average_time: number
}
```

this is an output of backend
```
    {
        "id": 1,
        "name": "qwerty",
        "weight": 2,
        "number_of_steps": 2
    }
```

however i can change it in backend

Reply:
changed state to status
output is for **Scenario** but this is **Result** type !!

Reply:
```
 {
        "id": "13f2ebca-14a9-4096-b520-9228073dd3fe",
        "team_name": "qaz",
        "team": "7c90d362-c443-40da-b428-fab41656dd0f",
        "scenario": "http://127.0.0.1:8000/api/v1/evaluation/scenarios/1/",
        "status": "DONE",
        "average_time": 12,
        "score": 100
    }
```